cozepy/websockets/audio/transcriptions/__init__.py

```
@@ -42,31 +65,86 @@ def _dump_without_delta(self):
     event_type: WebsocketsEventType = WebsocketsEventType.TRANSCRIPTIONS_CREATED
 
 
+# resp
+class TranscriptionsUpdatedEvent(WebsocketsEvent):
+    """配置更新成功
+
+    配置更新成功后，会返回最新的配置。
+    docs: https://www.coze.cn/open/docs/developer_guides/asr_event#3f842df1
+    """
+
+    class Data(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)